tests/module.py

```diff
@@ -242,6 +242,9 @@ def test_heapq(self):
   def test_hmac(self):
     self.assertOnlyIn((2.2, 3.0), detect("import hmac"))
 
+  def test_hmac_digest(self):
+    self.assertOnlyIn(3.7, detect("import hmac.digest"))
+
```

Comment:
Please move this test to "tests/function.py" around line 569, similar to:
```py
def test_digest_from_hmac(self):
  self.assertOnlyIn(3.7, detect("import hmac\nhmac.digest()"))
```

Reply:
Of course! Totally logical, I fix it.